pandas/tests/extension/test_numpy.py

```diff
@@ -199,6 +199,10 @@ class TestMethods(BaseNumPyTests, base.BaseMethodsTests):
     def test_value_counts(self, all_data, dropna):
         pass
 
+    @pytest.mark.xfail(reason="not working")
```

Comment:
do we have an issue here? why is this not working?

Reply:
can't use `PandasDtype(np.dtype('float'))` and `PandasDtype(np.dtype('object'))` in astype().

```
self = <PandasArray>
[(0,), (1,), (2,), (3,), (4,), (5,), (6,), (7,), (8,), (9,)]
Length: 10, dtype: object, dtype = PandasDtype('object'), copy = False

    def astype(self, dtype, copy=True):
>       return np.array(self, dtype=dtype, copy=copy)
E       TypeError: data type not understood
```

Reply:
Yes, this is covered by https://github.com/pandas-dev/pandas/issues/32028

Reply:
pls update the xfail message with this issue number